tests/test_cli_parser.py

```
@@ -3,63 +3,29 @@
-
-    run_incremental.assert_called_once()
-    run_full.assert_not_called()
-
-
 def test_cli_dispatches_full_by_default() -> None:
-    with (
-        patch("main.incremental_analysis.run_from_args") as run_incremental,
-        patch("main.full_analysis.run_from_args") as run_full,
-    ):
+    with patch("main.full_analysis.run_from_args") as run_full:
         main(["full", "--local", "/tmp/repo"])
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/repo"

(S108)